cmsplugin_articles_ai/models/articles.py

```diff
@@ -153,6 +153,9 @@ class Article(PublisherModel):
     modified_at = models.DateTimeField(
         _("last modified"), auto_now=True, editable=False,
     )
+    show_lead = models.BooleanField(
+        _("show lead"), default=True, help_text=_("Show lead text."),
```

Comment:
Still wouldn't add `show_lead`.. It just feels unnecessary. How about just not render it if it's empty? Or make the used template configurable. Having `show_lead` in model just feels unnecessary to me. I might be wrong though..